### PR TITLE
UserStory #25 - Display Review

### LINKED ISSUE #145: SP: 3 Create serializer for reviews GET method
Part of user story #25
Create additional serializer for getting all and by id.

### LINKED ISSUE #147: SP: 3 Create tests for display review
Part of user story #25

### LINKED ISSUE #169: SP: 3 Create GET method view for reviews
Part of user story #25

### PR DESCRIPTION
**Pull request by OleksiiDatsiuk**
I have implemented: 

- Serializer for displaying info about review
- View with GET method provided to get reviews for specified specialist
- Integration with flake8
- Tests for these methods

Closes #145 
Closes #169 
Closes #147

### CODE REVIEW

File: beauty/api/tests/test_GET_method_to_get_reviews.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+class TestReviewDisplayView(TestCase):
+    """This class represents a Test case and has all the tests."""
+
+    def setUp(self) -> None:
+        """This method adds needed info for tests."""
+        self.user = CustomUserFactory.create(is_active=True)
+        self.first_responder = CustomUserFactory(first_name="UserSpecialist")
+        self.second_responder = CustomUserFactory(first_name="UserSpecialist2")
+
+        self.review = ReviewFactory.create(from_user=self.user, to_user=self.first_responder)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_GET_method_to_get_reviews.py]: Why do you do it like that? Do you need to assign it to the same variable or can you just bulkcreate those reviews and then assign one of them to that variable.

File: beauty/api/tests/test_GET_method_to_get_reviews.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from .factories import CustomUserFactory, ReviewFactory
+from api.serializers.review_serializers import ReviewDisplaySerializer
+
+
+class TestReviewDisplayView(TestCase):
+    """This class represents a Test case and has all the tests."""
+
+    def setUp(self) -> None:
+        """This method adds needed info for tests."""
+        self.user = CustomUserFactory.create(is_active=True)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_GET_method_to_get_reviews.py]: no need in is_active=True, I have fixed that in factories, I believe

File: beauty/api/tests/test_GET_method_to_get_reviews.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                                           "api:review-get",
+                                           kwargs={"to_user": self.first_responder.id},
+                                       ),
+                                       )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data), 3)
+
+    def test_get_method_to_obtain_specified_review(self) -> None:
+        """Get review by its id."""
+        response = self.client.generic(method="GET",
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_GET_method_to_get_reviews.py:44]: self.client.get won't work?

**Review by therobotisnotatoy — APPROVED** [review]

File: beauty/api/tests/test_GET_method_to_get_reviews.py (hunk 1 above)
Comment (OleksiiDatsiuk) [reply beauty/api/tests/test_GET_method_to_get_reviews.py]: Oh, I have forgotten to change var names

File: beauty/api/tests/test_GET_method_to_get_reviews.py (hunk 2 above)
Comment (OleksiiDatsiuk) [reply beauty/api/tests/test_GET_method_to_get_reviews.py]: Got it

File: beauty/api/tests/test_GET_method_to_get_reviews.py (hunk 3 above)
Comment (OleksiiDatsiuk) [reply beauty/api/tests/test_GET_method_to_get_reviews.py:44]: Decided to use _client.generic_ cuz saw other using it

File: beauty/api/tests/test_GET_method_to_get_reviews.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+class TestReviewDisplayView(TestCase):
+    """This class represents a Test case and has all the tests."""
+
+    def setUp(self) -> None:
+        """This method adds needed info for tests."""
+        self.user = CustomUserFactory.create()
+        self.first_responder = CustomUserFactory(first_name="UserSpecialist")
+        self.second_responder = CustomUserFactory(first_name="UserSpecialist2")
+
+        ReviewFactory.create(from_user=self.user, to_user=self.first_responder)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_GET_method_to_get_reviews.py:22]: You can still batch create 3 of them =)

**Review by pazuzu-ua — APPROVED** [review]

**Pull request merged**